Sources/Commands/Utilities/SymbolGraphExtract.swift

```diff
@@ -48,12 +57,115 @@ public struct SymbolGraphExtract {
         outputRedirection: TSCBasic.Process.OutputRedirection = .none,
         outputDirectory: AbsolutePath,
         verboseOutput: Bool
+    ) throws {
+        if target.underlyingTarget is SwiftTarget {
```

Comment:
@abertelrud do you have any thoughts on what we should do here if the given target isn't a Swift or Clang? I suppose we could always fall back to attempting to use the swift symbol graph extract tool to match the current behavior. Throwing an error seems a little much since `swift package dump-symbol-graph` always runs against all the targets in a given package.

Reply:
Probably the right thing for now is to skip non-source targets.  In practice, today a source module target is always either a Swift or a Clang target (mixed targets aren't currently supported, nor are source module targets that use some other compiler driver).   Invoking the Swift symbol graph generator for other kinds of targets doesn't seem right... (alhough if that's the current behavior, we might need to keep it for at least tools versions 5.6 an earlier).